generators/base-workspaces/tasks.d.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2013-2025 the original author or authors from the JHipster project.
+ *
+ * This file is part of the JHipster project, see https://www.jhipster.tech/
+ * for more information.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import type { Merge } from 'type-fest';
+import type { TaskParamWithControl } from '../base/tasks.js';
+import type { Application as SimpleApplication, Source as SimpleSource } from '../base-simple-application/types.d.ts';
+import type { SimpleTaskTypes } from '../base-simple-application/tasks.js';
+
+export type TaskParamWithApplications<A, W, D> = TaskParamWithControl & {
+  applications: A;
+  deployment: D;
+  workspaces: W;
+};
+
+export type Tasks<S = SimpleSource, A = SimpleApplication[], W = any, D = any> = Merge<
```

Comment:
```suggestion
export type Tasks<A = SimpleApplication[], W = any, D = any> = Merge<
```

Source is not provided by workspaces.

Reply:
It does:
<img width="1001" alt="image" src="https://github.com/user-attachments/assets/cf8d7fb5-7e10-4bf7-a56a-44c1c5be9b99" />